connector/mqtt/latency-pub-test-prometheus/README.md

```diff
@@ -0,0 +1,63 @@
+# Latency pub test prometheus
+
+### Latency between publication MQTT and Kafka with Prometheus and Dojot
+
+A [prometheus](https://github.com/prometheus/prometheus) client to measure the latency between publication MQTT and when payload arrives in Apache Kafka for dojot.
+
+
+### **How works**
+
+Prometheus collects metrics from monitored targets by scraping metrics HTTP endpoints on these targets. When Prometheus scrapes your instance's HTTP endpoint, the client exposes metrics between the last collector to the current.
+
+Every time that this client receives a message from Kafka It store a latency, like describe below:
+
+ - The latency is measured with the diffence between final timestamp and start timestamp.
```

Comment:
- The latency is measured with the difference between final timestamp and start timestamp.

Reply:
?

Reply:
you must change diffence to difference